code/modules/clothing/spacesuits/chronosuit.dm

```diff
@@ -178,8 +179,6 @@
 						camera.remove_target_ui()
 			else
 				new_camera(user)
-	else
```

Comment:
this won't break chronosuits, right?

Reply:
This was done so Chrono suits will not break.